integ/util.go

```diff
@@ -196,3 +196,17 @@ func minimumContinueAsNewConfig() *iwfidl.WorkflowConfig {
 		ContinueAsNewThreshold: iwfidl.PtrInt32(1),
 	}
 }
+
+func getBackendTypes() []service.BackendType {
```

Comment:
This is a good idea to improve the code reuse. 
Let's return `[]service.BackendType{ service.BackendTypeTemporal, service.BackendTypeCadence}` if not specify for Temporal or Cadence. This is for making it easy for local test when you click the button to run the test directly. It doesn't supply the flag of cadence or temporal, but we want to do both if possible. 
https://github.com/indeedeng/iwf/blob/main/integ/flag.go#L11

Reply:
Actually nvm, they are default to true already.